### PR TITLE
Introduce activity grid, improve perf a bit

### PR DESCRIPTION
**Pull request by MrPepperoni**

### CODE REVIEW

File: src/core/collision_component.cpp [hunk 1]
```diff
@@ -59,9 +71,21 @@ void CollisionComponentLoader::BindValues()
...
 
+void CollisionComponentLoader::FillProperties( ComponentHolder& actor ) const
+{
+    bool dynradius = false;
+    if( mSetters != nullptr )
+    {
+        Json::GetBool( (*mSetters)["dynamic_radius"], dynradius );
+    }
+    ComponentLoader<CollisionComponent>::FillProperties( actor );
+    actor.Get<ICollisionComponent>()->SetDynamicRadius( dynradius );
```
Comment (HalalUr) [inline src/core/collision_component.cpp:86]: Is this really necessary? In theory using Bind( "radius", Bind("dynamic_radius".. will preserve order, and dynamic_radius will be the last that overrides the dyn_rad value. have you tried it?

File: src/render/actor_renderer.cpp [hunk 2]
```diff
@@ -75,23 +76,8 @@ bool isVisible( Actor const& actor, Camera const& camera )
...
-        {
-            if( i->second->GetScale() > f )
-            {
-                f = i->second->GetScale();
-            }
-        }
-        it = scaleMap.find( actor.GetId() );
-    }
-    float scale = it->second;
+    static RenderableRepo& renderables( RenderableRepo::Get() );
```
Comment (HalalUr) [inline src/render/actor_renderer.cpp:79]: renderableRepo (or mRenderableRepo if static imo it's kind of okay too)
static auto& renderableRepo(RenderableRepo::Get());

File: src/core/collision_component.cpp (hunk 1 above)
Comment (MrPepperoni) [reply src/core/collision_component.cpp:86]: yep, tried it. if there's no entry in the json for dyn radius, the setter is not called.

File: src/render/actor_renderer.cpp (hunk 2 above)
Comment (MrPepperoni) [reply src/render/actor_renderer.cpp:79]: Gonna change it in another PR, i have other changes here that i preferable don't want to conflict with.

**Pull request merged**